### ISSUE TITLE
Exception when adding OSGi aspect after upgrade to v0.12.

### ISSUE THREAD

**Issue by zaunerc** [ctron/package-drone#144]
I have upgraded from package drone version 0.11 to 0.12.1.20160211-1555. When adding the OSGi aspect I get an exception. Adding other aspects works so far.

Attached files:
- [stacktrace.txt](https://github.com/ctron/package-drone/files/174177/stacktrace.txt)
- [export-channel.zip](https://github.com/ctron/package-drone/files/174200/export-channel.zip)

**Comment by ctron** [comment]
That looks strange. Did you see the version number in the exception?

**Comment by zaunerc** [comment]
Yes. The problem is that the OSGi version of the xstream-1.4.8-sources bundle is erroneous:

![xstream-1 4 8-sources-osgi-version](https://cloud.githubusercontent.com/assets/14179675/13804012/8f0de548-eb49-11e5-8c3f-ce52a9a49690.PNG)

Maybe some OSGi related sanity checks should be run when importing artifacts into a channel with the OSGi aspect enabled. See e.g. the following screenshot:

![import_test](https://cloud.githubusercontent.com/assets/14179675/13804053/e93bcc6a-eb49-11e5-8207-664218061978.PNG)

I think the import test report should uncheck the xstream-1.4.8-sources bundle and tell the user that the version information in the OSGi manifest file is erroneous (like it did with the missing xmlpull javadoc and sources artifacts).

By the way. When adding the OSGi aspect **after** importing the artifacts you will get the following stacktrace: [import_after_aspects.txt](https://github.com/ctron/package-drone/files/175391/import_after_aspects.txt)

**Comment by ctron** [comment]
Ok, this I clearly a case then for the OSGi validator. I already detects and reports illegal bundle symbolic names. So the artifact would be uploaded, but not processed as OSGi bundle. And a validation error would be attached to the artifact and channel.

**Comment by ctron** [comment]
It would look like that now:

![pd013_fix_osgi1](https://cloud.githubusercontent.com/assets/202474/13810326/97ec9fd0-eb6f-11e5-961c-da1f6628d414.png)
![pd013_fix_osgi2](https://cloud.githubusercontent.com/assets/202474/13810327/97ed97c8-eb6f-11e5-8dae-9d933c8fb69a.png)

**Comment by zaunerc** [comment]
This looks perfectly fine to me. Thanks for coming up with a fix that quick :-)
